src/test/java/com/spencerwi/either/EitherFlatMapTest.java

```diff
@@ -0,0 +1,94 @@
+package com.spencerwi.either;
+
+import org.junit.jupiter.api.Test;
+
+class EitherFlatMapTest {
```

Comment:
These tests can be moved into `EitherTest`, since they're testing what will become "core functionality" for `Either`.